models/app.model.js

```diff
@@ -31,4 +31,48 @@ const fetchArticles = () => {
     })
 }
 
-module.exports = { readTopics, fetchArticlesbyId, fetchArticles }
\ No newline at end of file
+const fetchCommentsByArticleId = (article_id) => {
+    return db.query(`SELECT articles.article_id, comment_id, comments.votes, comments.created_at, comments.author, comments.article_id 
+    FROM articles 
```

Comment:
I can see the thinking behind what you have here and it's almost there! Remember, we want to select from the comments table and what foreign key connects the comments table and the articles table? The article_id, which we have access to 😉  how could we change this knowing this information?